apps/hash-graph/lib/graph/src/shared/identifier/time/temporal_axes.rs

```diff
@@ -0,0 +1,341 @@
+use serde::{Deserialize, Serialize};
```

Comment:
The location of these modules seems a bit strange to me. Are the axes identifiers?

Reply:
I moved it in [Make temporal_axes.rs a submodule of subgraph](https://github.com/hashintel/hash/pull/1854/commits/f53786a5c035df6e2e6ed4f914467ce773b52466)